src/abstractdataframe/selection.jl

```diff
@@ -54,9 +54,9 @@ const TRANSFORMATION_COMMON_RULES =
     6. vectors or matrices containing transformations specified by the `Pair` syntax
        described in points 2 to 5
     8. a function which will be called with a `SubDataFrame` corresponding to each group;
-       this form should be avoided due to its poor performance unless a very large
-       number of columns are processed (in which case `SubDataFrame` avoids excessive
-       compilation)
+       this form should be avoided due to its poor performance unless the number of groups
```

Comment:
looks good. Could you please also sync the manual entry that has the same passage. Thank you!

Reply:
bump

Reply:
bump :)

Reply:
Woops. Done!